core/src/main/java/org/elasticsearch/index/query/MissingQueryBuilder.java

```diff
@@ -78,15 +102,6 @@ public boolean nullValue() {
     }
 
     /**
-     * Should the missing filter include documents where the field doesn't exist in the docs.
-     * Defaults to <tt>true</tt>.
-     */
-    public MissingQueryBuilder existence(boolean existence) {
```

Comment:
The reason was that currently validate() needs to check if both values are `false` (only illegal combination), can't do that in one setter so I moved both to ctor. I made the ctor args optional (Boolean) because I didn't want users to have to specify both if they just want to set one of the values, they have a relation but they really are two separate settings, which is also why I didn't choose to have a setter that takes two arguments.